documentation.html

```diff
@@ -39,8 +42,208 @@ <h6>Components</h6>
                 <a href="" class="link">Toggle Button</a>
             </aside>
             <div class="container">
-                <div class="sub_container"></div>
+                <div class="sub_container">
+                    <!------------------------------------------------------- Alert section ------------------------------------------------------->
+
+                    <section id="alert">
+                        <h1>Alert</h1>
+                        <p>Alerts are used to communicate a state that affects a system, feature or page.</p>
+                        <h4>Example</h4>
+                        <div class="element_container column">
+                            <div class="alert alert--success" role="alert">
+                                <div class="alert__icon">
+                                    <i class="fas fa-check"></i>
+                                </div>
+                                <div class="alert__text">
+                                    <span>Data uploaded to the server. Fire on!</span>
+                                </div>
+                            </div>
+                        </div>
+                        <h4>Alert theme</h4>
+                        <p>Alerts are available in 4 different color scheme show case any system alert to users</p>
+                        <div class="element_container column">
+                            <div class="alert alert--success" role="alert">
+                                <div class="alert__icon">
+                                    <i class="fas fa-check"></i>
+                                </div>
+                                <div class="alert__text">
+                                    <span>Data uploaded to the server. Fire on!</span>
+                                </div>
+                            </div>
+                            <div class="alert alert--info" role="alert">
+                                <div class="alert__icon">
+                                    <i class="fas fa-check"></i>
+                                </div>
+                                <div class="alert__text">
+                                    <span>Update is ready to be installed.</span>
+                                </div>
+                            </div>
+                            <div class="alert alert--warning" role="alert">
+                                <div class="alert__icon">
+                                    <i class="fas fa-check"></i>
```

Comment:
probably an oversight. try to change the icon for warning and info, currently, all are showing 'check'.

Reply:
Yeah. that's on my to-do list. Thanks